examples/node_pool/main.tf

```diff
@@ -60,7 +60,7 @@ module "gke" {
       accelerator_count  = 1
       accelerator_type   = "nvidia-tesla-a100"
       gpu_partition_size = "1g.5gb"
-      image_type         = "COS"
+      image_type         = "COS_CONTAINERD"
```

Comment:
I believe our default is cos_containerd so we can skip setting this.
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/93599618529ba532ad5b118b59497f502a020d4e/cluster.tf#L269

Reply:
Sure, I'll remove it from all the examples in this commit.